webpack.config.js

```diff
@@ -349,7 +375,7 @@ module.exports = (env, args = {}) => {
         vendor: {
           test: /[\\/]node_modules[\\/]/,
           name: 'vendor',
-          chunks: chunk => !['polyfills.ie11', 'polyfills'].includes(chunk.name), // Excludes node modules which are required by IE11 polyfills
+          chunks: chunk => !['polyfills'].includes(chunk.name), // Excludes node modules which are required by IE11 polyfills
```

Comment:
I think this can be removed. Since we will only have a few polyfills (hopefully) and we won't be able to distinguish modern browsers as easy as IE11 we can just bundle the polyfills with vendor/app.